.idea/misc.xml

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="ExternalStorageConfigurationManager" enabled="true" />
+  <component name="JavaScriptSettings">
```

Comment:
?

Reply:
My IntelliJ's been adding that in for a while, no clue why it keeps doing that (maybe it's because of plugins?).